docs/docsite/rst/plugins/lookup.rst

```diff
@@ -10,21 +10,23 @@ Lookup Plugins
 Lookup plugins are an Ansible-specific extension to the Jinja2 templating language. You can use lookup plugins to access data from outside sources (files, databases, key/value stores, APIs, and other services) within your playbooks. Like all :ref:`templating <playbooks_templating>`, lookups execute and are evaluated on the Ansible control machine. Ansible makes the data returned by a lookup plugin available using the standard templating system. You can use lookup plugins to load variables or templates with information from external sources.
 
 .. note::
-   - Lookups are executed with a working directory relative to the role or play,
-     as opposed to local tasks, which are executed relative the executed script.
-   - Pass ``wantlist=True`` to lookups to use in Jinja2 template "for" loops.
-   - By default, lookup return values are marked as unsafe for security reasons. If you trust the outside source your lookup accesses, pass ``allow_unsafe=True`` to allow Jinja2 templates to evaluate lookup values.
+   - Pass ``wantlist=True`` to lookups to use in Jinja2 template "for" loops. See :ref:`query` for more details.
 
 .. warning::
-   - Some lookups pass arguments to a shell. When using variables from a remote/untrusted source, use the `|quote` filter to ensure safe usage.
-
+   - By default, lookup return values are marked as unsafe for security reasons. If you trust the outside source that your lookup accesses, pass ``allow_unsafe=True`` to allow Jinja2 templates to evaluate lookup values.
+   - Some lookups pass arguments to a shell. When using variables from a remote/untrusted source, use the ``|quote`` filter to ensure safe usage.
 
 .. _enabling_lookup:
 
 Enabling lookup plugins
 -----------------------
 
-Ansible enables all lookup plugins it can find. You can activate a custom lookup by either dropping it into a ``lookup_plugins`` directory adjacent to your play, inside the ``plugins/lookup/`` directory of a collection you have installed, inside a standalone role, or in one of the lookup directory sources configured in :ref:`ansible.cfg <ansible_configuration_settings>`.
+Ansible enables all lookup plugins it can find. You do not need to maintain a list of enabled lookup plugins. To activate a custom lookup, place it in one of four locations:
```

Comment:
this is not true anymore, it enables all those in 'configured paths' but for collections you have to do extra work

Reply:
oh, interesting, so how do I enable a lookup plugin from a collection?

Reply:
Simplest way, use fqcn and it will 'load on demand', though I believe we also added support with 'collections:' keywords to use short names, but that i need to confirm (nitz should know top of his head).

 Of course in both cases collection needs to be in collection search path.

Reply:
some of my fixes will allow listing filters and plugins from collections also via `ansible-doc -l` but a bit stuck on that right now

Reply:
Add note for collections:
```
 - If the lookup is part of a collection, the collection itself must be installed in the discoverable paths for collections (ref?)
 ```